src/olympia/addons/cron.py

```diff
@@ -180,29 +179,22 @@ def deliver_hotness():
     threshold = 250 if addon type is theme, else 1000
     hotness = (a-b) / b if a > threshold and b > 1 else 0
     """
-    frozen = set(f.addon_id for f in FrozenAddon.objects.all())
-    averages = get_averages_by_addon_from_bigquery(today=date.today())
-    addons = (
-        Addon.objects.filter(guid__in=averages.keys())
-        .filter(status__in=amo.REVIEWED_STATUSES)
-        .exclude(id__in=frozen)
+    frozen_guids = list(set(fa.addon.guid for fa in FrozenAddon.objects.all()))
+    averages = get_averages_by_addon_from_bigquery(
+        today=date.today(), exclude=frozen_guids
     )
-
-    for addon in addons:
-        average = averages.get(addon.guid)
-        this = average['avg_this_week']
-        three = average['avg_three_weeks_before']
-
-        # Update the hotness score but only update hotness if necessary. We
-        # don't want to cause unnecessary re-indexes.
-        threshold = 250 if addon.type == amo.ADDON_STATICTHEME else 1000
-        if this > threshold and three > 1:
-            hotness = (this - three) / float(three)
-            if addon.hotness != hotness:
-                addon.update(hotness=hotness)
-        else:
-            if addon.hotness != 0:
-                addon.update(hotness=0)
-
-        # Let the database catch its breath.
-        time.sleep(10)
+    create_chunked_tasks_signatures(
+        update_addon_hotness, averages.items(), chunk_size
+    ).apply_async()
+
+    # Reset add-ons that won't be returned by BigQuery.
+    addon_ids = (
+        Addon.objects.filter(status__in=amo.REVIEWED_STATUSES)
+        .filter(hotness__gt=0)
+        .exclude(guid__in=averages.keys())
+        .no_transforms()
```

Comment:
nit: the transforms won't be executed with a `values_list` so it's not needed.

Reply:
TIL!